sk0-front/src/app/auth/store/auth.reducer.spec.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  LoginRequest,
+  LoginResponse,
+  RegisterRequest,
+  RegisterResponse,
+  loginFailure,
+  loginStarted,
+  loginSuccess,
+  logoutAction,
+  registerFailure,
+  registerStarted,
+  registerSuccess
+} from "./auth.actions";
+import { AuthState, authStore } from "./auth.reducer";
+
+describe("Auth Reducers", () => {
+  const mockInitialAuthState: AuthState = {
```

Comment:
Why is that a 'mock' value? I think this is the actual value and not a mock of it. If you are testing the reducer functions it should not be mocked.